Tests/Packet++Test/main.cpp

```diff
@@ -8290,6 +8290,47 @@ PTF_TEST_CASE(BgpLayerEditTest)
 } // BgpLayerEditTest
 
 
+PTF_TEST_CASE(ResizeLayerTest)
+{
+	// Creating a packet
+	Packet packet(1500);
+		
+	uint8_t payload[] = { 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0xa };
+	PayloadLayer payloadLayer(payload, 10, true);
+	PTF_ASSERT(packet.addLayer(&payloadLayer), "Adding payload layer failed");

```

Comment:
please don't use `PTF_ASSERT`, instead there are newer asserts to use like `PTF_ASSERT_EQUAL`, `PTF_ASSERT_TRUE`, etc. Please take a look at:

https://github.com/seladb/PcapPlusPlus/blob/0f456fc2a30e7f780db413d391412f3d67df94e2/Tests/Packet%2B%2BTest/main.cpp#L7679